cpg_workflows/large_cohort/combiner.py

```diff
@@ -95,24 +86,21 @@ def run(
         params.setdefault('use_genome_default_intervals', True)
     else:
         raise ValueError(
-            'Either combiner/intervals must be set, or workflow/sequencing_type '
-            'must be one of: "exome", "genome"'
+            'Either combiner/intervals must be set, or workflow/sequencing_type ' 'must be one of: "exome", "genome"',
```

Comment:
```suggestion
            'Either combiner/intervals must be set, or workflow/sequencing_type must be one of: "exome", "genome"',
```

Reply:
I find / replaced these all locally in 17967aa